packages/core/core/src/Parcel.js

```diff
@@ -75,23 +75,6 @@ export default class Parcel {
       throw new Error('Could not find a .parcelrc');
     }
 
-    this.farm = await WorkerFarm.getShared(
-      {
-        config,
-        cliOpts: this.options.cliOpts,
-        env: this.options.env
-      },
-      {
-        workerPath: require.resolve('./worker')
-      }
-    );
-
-    this.runPackage = this.farm.mkhandle('runPackage');
-  }
-
-  async run() {
-    await this.init();
-
     this.bundlerRunner = new BundlerRunner({
```

Comment:
Not sure if bundlerRunner should be part of init, as it's not being used in the parcel/register hook

Reply:
Ah, I suggested he fix it this way since I noticed that's what @devongovett was doing in his reporters branch (https://github.com/parcel-bundler/parcel/pull/2632/files#diff-bf9c3feeacb4c7e393721871916cf9f2R65)